packages/block-library/src/template-part/edit/inner-blocks.js

```diff
@@ -14,7 +14,7 @@ export default function TemplatePartInnerBlocks() {
 	);
 	return (
 		<InnerBlocks
-			__experimentalBlocks={ blocks }
+			value={ blocks }
```

Comment:
So, this PR makes this API stable. Do you think we should do it now or still leave it experimental for some time?

Reply:
Great question! I'll defer to you and @epiqueras on this one, but this essentially just changes it to use the same API that `BlockEditorProvider` already uses. Since that API is stable, it seems kind of weird to mark this instance of the API as unstable while keeping that one stable, even though the code under the hood is the same. 

On the other hand, the main new changes are just to support controlled inner blocks, which is still pretty fresh.

Whatever works for me :)

Reply:
I think it's mature enough now.

Reply:
Let's follow up with some docs in the component's README then.